feature/experimental/ancxplaceholder/readme.md

```diff
@@ -0,0 +1,216 @@
+# RT-1.35: ancx placeholder test
+
+## Summary
+This is a placeholder test to cover for all anCX OCpaths that are not already covered in any FNT.
+
+
+### Config paths
+* /components/component/integrated-circuit/utilization/resources/resource/config/name
+* /components/component/linecard/utilization/resources/resource/config/name
```

Comment:
Hi, needs to be in this format to pass CI checks / parsed by the intended tools:

https://github.com/openconfig/featureprofiles/blob/main/doc/test-requirements-template.md#openconfig-path-and-rpc-coverage

Reply:
> Hi, needs to be in this format to pass CI checks / parsed by the intended tools:
> 
> https://github.com/openconfig/featureprofiles/blob/main/doc/test-requirements-template.md#openconfig-path-and-rpc-coverage

PTAL now